docs/getting-started/apple/screenshots.md

```diff
@@ -1,8 +1,258 @@
+
+The positive side of this approach is that your screenshots will all be crisp and correctly sized, with readable text.
+
+### Scaled Screenshots via iTunes Connect
+
+With the [iTunes Connect screenshot update](https://developer.apple.com/news/?id=08082016a) from August 8th, 2016, you can now use one set of screenshots for all available devices and screenshots. iTunes Connect will automatically scale the images for you, so that each device renders the same exact image.
+
+While this is convenient, this approach has the same problems as the device frame approach: The screenshots don't actually show how the app looks on the user's device. It's a valid way to start though, since you can gradually overwrite screenshots for specific languages and devices.
+
+# Capture Screenshots Automatically
+
+_snapshot_ works with _fastlane_ to automate the process of capturing screenshots of your app. It allows you to:
+
+- Capture hundreds of screenshots in multiple languages on all simulators
+- Do something else while the computer takes the screenshots for you
+- Configure it once, and store the configuration so anyone on the team can run it
+- Generate a beautiful web page showing all screenshots on all devices. This is perfect to send to Q&A, marketing, or translators for verification
+- Avoid having loading indicators in your App Store screenshots by intelligently waiting for network requests to be finished
+
+After _snapshot_ successfully captures all the screenshots, it will show you a beautiful HTML page to summarize all the screens that were captured:
+
+![HTML summary page](assets/htmlPagePreviewFade.jpg)
+
+## Getting Started Using UI Tests
+
+_snapshot_ uses the capabilities of Apple's UI Tests to drive interactions with your app. To get familiar with writing UI Tests, check out the following introductions:
+
+- [WWDC 2015 Introduction to UI Tests](https://developer.apple.com/videos/play/wwdc2015-406/)
+- [A first look into UI Tests](http://www.mokacoding.com/blog/xcode-7-ui-testing/)
+- [UI Testing in Xcode 7](http://masilotti.com/ui-testing-xcode-7/)
+- [HSTestingBackchannel : ‘Cheat’ by communicating directly with your app](https://github.com/ConfusedVorlon/HSTestingBackchannel)
+- [Automating App Store screenshots using fastlane snapshot and frameit](https://tisunov.github.io/2015/11/06/automating-app-store-screenshots-generation-with-fastlane-snapshot-and-sketch.html)
+
+To jump-start your UI tests, you can use the UI Test recorder. By interacting with your app through the simulator while using it, Xcode will generate UI Test code for you. You can find the red record button on the bottom of the screen, and there's more information in [this blog post](https://krausefx.com/blog/run-xcode-7-ui-tests-from-the-command-line)
```

Comment:
The last part `and there's more information in` seems a bit off, maybe it should be its own sentence?